articles/tokens/access_token.md

```diff
@@ -1,49 +1,115 @@
+* The [?? endpoint in the authentication API](??) when using Client Credentials Grant will issue an `access_token`
 * Check the [List of tutorials](/tutorials) to see how to make calls to libraries for other languages/SDKs.
 
 ## How to control contents of access token
 
-It is not possible at this time to control the contents of the Auth0 access token.
+The contents of the `access_token`, specifically the claims contained within it, are controlled by the use of a parameter called `scope` which is passed to the authentication functions mentioned above.  For example, the call to the Lock widget’s `.show` function can specify optional authentication parameters as follows:
+
+?? Is this still true - can we use lock ??
+
+```
+lock.show({
+   responseType: ‘token’,
+   authParams: {
+      scope: ‘appointments contacts’
+    }
+ });
+```
+The above sample, specifying `appointments contacts` will result in a JWT with claims for accessing appointments and contacts. The responseType should be `token` for Implicit Grant flow and `code` for Authorization Code Grant flow.
+
+The scope of the `access_token` JWT can also be altered via Rules, through the context.jwtConfiguration.scopes object as documented [here](https://github.com/auth0/docs/blob/32033877180affa26233b8f65cb28bd532514eab/articles/rules/index.md#context) ?? Should be still suggest this. Also AFAIK rules won't work anymore with the new endpoint ??
+
+There is a [sample for altering scopes in a Rule](https://github.com/auth0/rules/blob/dff2a3e72f01d33af3086414be7cf115b19eea0c/rules/custom-scopes.md)
+
+Additional information on the `access_token` is [here](/jwt)
+
+A valid JWT can be pasted into the [jwt.io website](https://jwt.io) to view the contents of the JWT.
+
+A [blog entry on JWT](https://auth0.com/blog/2015/07/21/jwt-json-webtoken-logo/) provides an explanation of why it is getting to be more popular.
+
+[Additional samples](https://github.com/auth0/auth0.js) show use of the auth0.js library.
 
 ## Validity
 
-The Auth0 access token remains valid for 24 hours.
+In the case of the [authorization code grant](/api-auth/grant/authorization-code) and the [implicit code grant](/api-auth/grant/implicit) flows the `access_token` is valid for ???
+
+In the case of the [client credentials flow](/api-auth/grant/client-credentials), the `access_token` is valid for 8000 seconds (roughly 2.25 hours) by default.  !!The expiration of this token can be set in the `Apps/APIs` -> Settings screen using the `JWT expiration` field!!
+
+The validity period of the token can be configured per [Resource Server](??)
 
 ## Renewing the token
 
-There is currently no way to renew the Auth0 access token without triggering authentication by the user.  To get a new access token, simply follow the process for getting an access token as described above in the How to Get an Access Token section.
+?? How do we get a refresh token ??
```

Comment:
Yes, by requesting offline_access, you will get a refresh token that can refresh an access token.  This is currently not implemented correctly, but it will be.   Issuing of refresh tokens will be at the discression of the resource server.  If the refresh server disables this, no refresh token will be issued, and the client will have to go through a /authorize flow again to obtain a new token.